### PR TITLE
[Enhancement] Editing allowed in job summary

### LINKED ISSUE #415: [Enhancement] editing allowed in job summary

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closes #415

### CODE REVIEW

**Comment by JavaLavaMT** [comment]
screen shot 2019-01-08 at 11 58 19 am
screen shot 2019-01-08 at 11 58 39 am

**Comment by James1x0** [comment]
@JavaLavaMT You need to swap descrip editor for https://github.com/associatedemployers/granite/pull/446

**Comment by JavaLavaMT** [comment]
image

**Comment by JavaLavaMT** [comment]
image


image

File: app/routes/account/job-description/edit.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import Route from '@ember/routing/route';
+import RSVP from 'rsvp';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+import add from 'granite/mixins/route-abstractions/add';
+
+export default Route.extend(add, {
```
Comment (James1x0) [inline app/routes/account/job-description/edit.js]: Why even include it if you don't use it? Please use the `edit` abstraction and call super on L15

File: app/controllers/account/job-description/edit.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+import addEdit from 'granite/mixins/controller-abstractions/add-edit';
+import { jobCategories } from 'granite/config/statics';
+
+
```
Comment (James1x0) [inline app/controllers/account/job-description/edit.js]: Extra line break here

File: app/routes/account/job-description/edit.js (hunk 1 above)
Comment (JavaLavaMT) [reply app/routes/account/job-description/edit.js]: ✔

File: app/controllers/account/job-description/edit.js (hunk 2 above)
Comment (JavaLavaMT) [reply app/controllers/account/job-description/edit.js]: ✔

**Pull request merged**